src/pages/Address/RgbppAssets.tsx

```diff
@@ -68,6 +59,8 @@ const AssetItem: FC<{ cell: LiveCell }> = ({ cell }) => {
   let attribute = null
   let detailInfo = null
 
+  const utxo = `${cell.txHash.slice(0, 4)}...${cell.txHash.slice(-3)}:${cell.cellIndex + 1}`
```

Comment:
Why is `cell.cellIndex` increased by 1

Reply:
How about using https://github.com/nervosnetwork/ckb-explorer-frontend/blob/5b9f6a00ac296fe90dc9a71643065312c5bcd7db/src/components/EllipsisMiddle/index.tsx#L6 to shrink the string responsively.